.gitignore

```diff
@@ -1,2 +1,5 @@
 .stack-work/
 example-servant-minimal.cabal
+
+# Editor files:
+*~
```

Comment:
This should be in your global gitignore.

Reply:
Oh, good to know - I didn't realize there was a global .gitignore, will use that.